src/core.jl

```diff
@@ -15,11 +15,19 @@ struct Tape
     Tape(N::Int) = new(Vector{Any}(undef, N))
 end
 function show(io::IO, t::Tape)
-    if length(t) == 0
-        println(io, "Empty tape.")
+    n = length(t)
+    if n == 0
```

Comment:
This clause isn't necessary anymore, since the behaviour is the same as below. You can just change the summary `println` to a `print`, then add a newline at the beginning of the first `print` in the loop and remove the last line of the loop.

Reply:
Looking better?